README.md

```diff
@@ -42,6 +42,7 @@
   "type": "module",
   "source": "src/foo.js",             // your source code
   "exports": {
+    "types": "./dist/foo.d.ts",       // optional; only needed if authoring in TypeScript
```

Comment:
```suggestion
```

Sorry, I didn't realize we weren't previously documenting `"types"` in that example. That's my fault for suggesting this in the issue.

FWIW, with `--generateTypes`, Microbundle can create `.d.ts` files from the source, utilizing JSDoc.

Since `--generateTypes` defaults to `false`, and authoring in TS isn't a given, this was likely missing here to avoid references to files that may not exist.